src/events/cmd/main.go

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright © 2020, Oracle and/or its affiliates. All rights reserved.
+ * The Universal Permissive License (UPL), Version 1.0
+ */
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"mushop/events"
+
+	"github.com/go-kit/kit/log"
+	stdopentracing "github.com/opentracing/opentracing-go"
+	zipkin "github.com/openzipkin/zipkin-go-opentracing"
+	"golang.org/x/net/context"
+)
+
+const (
+	ServiceName = "events"
+)
+
+func main() {
+	var (
+		port = flag.String("port", "8080", "Port to bind HTTP listener")
+		zipk = flag.String("zipkin", os.Getenv("ZIPKIN"), "Zipkin address")
```

Comment:
we don't use this, right?

Reply:
not yet, but is consistent with the other `go` services. So would be simple to add/configure zipkin if desired